### DISCUSSION TITLE
GoPro Hero 7 Docs Linking to Hero 4 Docs

### DISCUSSION THREAD

**Discussion by chrisfoster121** [KonradIT/goprowifihack#265] [Q&A]
Hello, [The docs for the GoPro Hero 7](https://github.com/KonradIT/goprowifihack/blob/master/HERO7/README.md) include a[ link for live streaming](https://github.com/KonradIT/goprowifihack/blob/master/HERO4/Livestreaming.md). The link for live streaming is documentation for the GoPro Hero 4. Is this a mistake or does this actually work on the Hero 7 Black?

**Comment by KonradIT** [comment]
It does. Hero4-10 livestreaming is pretty much the same (`udp://10.5.5.9:8554`, but command for starting might be different across generations, Hero4-7 is the same).
